.isort.cfg

```diff
@@ -0,0 +1,2 @@
+[settings]
+known_first_party=qcore,empirical,nshmdb,IM_calculation,mera
```

Comment:
This list is non-exhaustive. You can just add packages here when you need to to make them first party.